framework/global/internal/interactive.h

```diff
@@ -47,7 +50,8 @@ class Interactive : public IInteractive
     void message(Type type, const std::string& title, const std::string& text) const override;
 
     // files
-    io::path selectOpeningFile(const std::string& title, const std::string& dir, const std::string& filter) override;
+    io::path selectOpeningFile(const QString& title, const QString& dir, const QString& filter) override;
```

Comment:
Maybe change the return value to QString too?

Reply:
I think using an alias is a good idea, but it's really worth making `QString` in essence. Although in the future, when we switch to C++17, I think it will be better to use `std::filesystem::path`